tests/exec_test/exec_test.go

```diff
@@ -0,0 +1,331 @@
+					Replicas: func(i int32) *int32 { return &i }(3),
+					Selector: &metav1.LabelSelector{
+						MatchLabels: map[string]string{
+							"app": "nginx",
+						},
+					},
+					Template: v1.PodTemplateSpec{
+						ObjectMeta: metav1.ObjectMeta{
+							Labels: map[string]string{
+								"app": "nginx",
+							},
+						},
+						Spec: v1.PodSpec{
+							ServiceAccountName: "litmus",
+							Containers: []v1.Container{
+								{
+									Name:  "nginx",
+									Image: "nginx:latest",
+									Ports: []v1.ContainerPort{
+										{
+
+											ContainerPort: 80,
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			}
+			Expect(err).ShouldNot(
+				HaveOccurred(),
+				"while building deployment nginx in namespace litmus",
+			)
+
+			By("creating above deployment")
+			_, err := client.AppsV1().Deployments("litmus").Create(deployment)
+			Expect(err).To(
+				BeNil(),
+				"while creating deployment nginx in namespace litmus",
+			)
+		})
+	})
+
+	When("Create Pod-delete chaosExperiment in litmus Namespace", func() {
+		It("should create a CustomResource ChaosExperiment Pod-delete", func() {
+
+			By("building a chaosExperiment")
+			ChaosExperiment := &v1alpha1.ChaosExperiment{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "pod-delete",
+					Namespace: "litmus",
+					Labels: map[string]string{
+						"litmuschaos.io/name": "kubernetes",
+					},
+				},
+				Spec: v1alpha1.ChaosExperimentSpec{
+					Definition: v1alpha1.ExperimentDef{
+
+						Args:    []string{"-c", "ansible-playbook ./experiments/chaos/pod_delete/test.yml -i /etc/ansible/hosts -vv; exit 0"},
+						Command: []string{"/bin/bash"},
+
+						ENVList: []v1alpha1.ENVPair{
+							{
+								Name:  "ANSIBLE_STDOUT_CALLBACK",
+								Value: "default",
+							},
+							{
+								Name:  "TOTAL_CHAOS_DURATION",
+								Value: "15",
+							},
+							{
+								Name:  "CHAOS_INTERVAL",
+								Value: "5",
+							},
+							{
+								Name:  "LIB",
+								Value: "",
+							},
+						},
+						Image: "",
+						Labels: map[string]string{
+							"name": "pod-delete",
+						},
+					},
+				},
+			}
+			Expect(err).ShouldNot(
+				HaveOccurred(),
+				"while building pod-delete chaosExperiment",
+			)
+
+			By("creating above chaosExperiment")
+			_, err = clientSet.ChaosExperiments("litmus").Create(ChaosExperiment)
+			Expect(err).To(
+				BeNil(),
+				"while creating chaosExerpiment Pod-delete in namespace litmus",
+			)
+		})
+	})
+	When("Creating ChaosEngine to trigger chaos-executor", func() {
+		It("should create a runnerPod and Service ", func() {
+
+			By("building a ChaosEngine")
+			chaosEngine := &v1alpha1.ChaosEngine{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "engine-nginx",
+					Namespace: "litmus",
+				},
+				Spec: v1alpha1.ChaosEngineSpec{
+					Components: v1alpha1.ComponentParams{
+						Runner: v1alpha1.RunnerInfo{
+							Type:  "go",
+							Image: "litmuschaos/chaos-executor:ci",
```

Comment:
instead of using ci image here you have to use the builded image of current changes